config/services.yaml

```diff
@@ -38,3 +38,13 @@ services:
     arguments:
       - '@security.helper'
       - "%logout_redirect_url%"
+
+  Surfnet\StepupRa\RaBundle\EventListener\AuthenticatedUserListener:
+
+  Surfnet\StepupRa\RaBundle\EventListener\ExplicitSessionTimeoutListener:
```

Comment:
Are these entries needed? Autoconfigure/autowire should pick this up without issue

Reply:
Yes, without it won't work as the  Surfnet\StepupRa\RaBundle\* is not in the search path